lib/src/exceptions.dart

```diff
@@ -25,13 +25,16 @@ class GQLError {
 
   /// The path of the field in error.
   final List<dynamic> path;
+  
+  final Map<String, dynamic> extensions;
```

Comment:
Plase a comment explaining the variable on the  line directly above the declaration. Most editors can process those comments and automatically generate documentation from it.